pkg/extensions/types.go

```diff
@@ -123,11 +125,65 @@ type IpClaimSpec struct {
 	Link     string `json:"link" protobuf:"bytes,10,opt,name=link"`
 }
 
+type IpClaimPool struct {
+	unversioned.TypeMeta `json:",inline"`
+
+	// Standard object metadata
+	Metadata api.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	Spec IpClaimPoolSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+}
+
+func (e *IpClaimPool) GetObjectKind() unversioned.ObjectKind {
+	return &e.TypeMeta
+}
+
+func (e *IpClaimPool) GetObjectMeta() meta.Object {
+	return &e.Metadata
+}
+
+type IpClaimPoolSpec struct {
+	Range string `json:"range" protobuf:"bytes,10,opt,name=range"`
```

Comment:
i think it is not entirely correct, this is cidr, and range should have first and last items. Thus if first and last is not specified - we will use whole cidr